cf/api/organizations/organizations.go

```diff
@@ -71,6 +72,26 @@ func (repo CloudControllerOrganizationRepository) FindByName(name string) (org m
 	return
 }
 
+func (repo CloudControllerOrganizationRepository) GetOrganizationQuotaUsage(orgGuid string) (models.QuotaUsage, error) {
+	url := fmt.Sprintf("%s/v2/organizations/%s/quota_usage", repo.config.ApiEndpoint(), orgGuid)
+
+	quotaUsageResource := new(resources.QuotaUsageResource)
+	err := repo.gateway.GetResource(url, quotaUsageResource)
+	if err != nil {
+		return models.QuotaUsage{}, err
+	}
+
+	return models.NewQuotaUsage(quotaUsageResource.Entity.Name,
```

Comment:
I feel better call ToFields  or similar  function here , instead of creating NewQuotaUsage